modules/graphics/weston.ini.nix

```diff
@@ -9,6 +9,8 @@
 
       [shell]
       locking=false
+      background-image=${./assets/wallpaper.jpg}
+      background-type=scale
```

Comment:
I recommend `scale-crop` instead. It will handle different sized displays better.

Reply:
i tried that, the problem is it was a bit overzealous in cropping
:)